### ISSUE TITLE
[Bug Report][3.5.14] AutoComplete @update:search triggered on click

### ISSUE THREAD

**Issue by augustoeliezer** [vuetifyjs/vuetify#19543]
### Environment
**Vuetify Version:** 3.5.14
**Vue Version:** 3.4.21
**Browsers:** Edge 123.0.0.0
**OS:** Windows 10

### Steps to reproduce
Choose one in autocomplete
Click outside autocomplete
Search occurs two/three times when its have no focus

### Expected Behavior
this event might be only triggered on type in search field

### Actual Behavior
Its triggered on click in autocomplete, on choose one item and when clicking outside the autocomplete

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqNU8Fu2zAM/RXCl7RonGTpzXCCDrlupx2bHlSZSYTJkiHR6QbD/z5KdhfbSYte4ug9kk96JJ+bxDu5/F5Vi3ONSZbkhGWlBeF2bwDycyqqKv6NB2kNCWXQgdTC+80+Kf+k633SR8QYZ9/+H/skPQC6ojVZaVkICYcUQKb4AqGwVp7EPhmzBRqv6C/TIVtImgZo8Yqa6R/hOyVJkUYmdydrPYI1OI04C6eEIY6xNWl+aHEVkZa2iBLxO6XD7dN3HXOTPQtdB1Zd1T6pAlNj00Lwy0fMU10xiJlH4eSJk0/CFBp/oZCncZVtvhzbO+wFU+Nm3O7OKxl4klrJ36zUSe5sbQg2sOJe7zQKA6zNOEQiFOakr0j9DK5l0DQQ/YO2HVWKzFCybaelwnk4Y18aucmZEa22lzkAZSAMXL5k+EZg8AJ4JDx3CIbufpDQP8lKWTsP9GZ59kr0V9H5cuzRp0/t2H4Bu/UMvY77mS8He8tHL52qiJ2kuuKlMEfuJPl+UVVZWUfQgMMDtHBwtoQZr/8skKzgqW/OJkTcre4v+Hgapmzc2R5/bkwGs4vFszmoIoNVO28gMISeeuwbtHO+zTu67uH1GH7s4UdoX1gzqB5qI0lZA4N1uIv7lYEnp8zxHprOvMHFFzHi4aEjws2txoW2xy41vqcNlnYmsmVJOze11vHn5R+9xovK)

### Other comments
```ts
<template>
  <v-app>
    <v-container class="mx-2">
      <v-row>
        <v-col>
          <v-autocomplete
            :items="list"
            density="compact"
            label="Label"
            title="Choose one"
            variant="outlined"
            v-model="model"
            item-title="n"
            item-value="id"
            hide-no-data
            @update:search="handleSeach"
          ></v-autocomplete>
        </v-col>
        <v-col>
          <v-btn @click="searchCount = 0">Clean Search Count</v-btn>
        </v-col>
        <v-col>Model: {{ model }} Search Count: {{ searchCount }}</v-col>
      </v-row>
      <v-row>
        <v-col>
          <ol>
            <li>Choose one in list</li>
            <li>Click outside autocomplete</li>
            <li>Search occurs two times</li>
          </ol>
        </v-col>
      </v-row>
    </v-container>
  </v-app>
</template>

<script setup lang="ts">
  import { ref } from 'vue'
  const model = ref(0)
  const searchCount = ref(0)
  const list = ref([{n: 'Choose one', id: 0},{ n: 'test', id: 1 }, { n: 'test2', id: 2 }, { n: 'test3', id: 3 }])

  function handleSeach(value: string) {
    searchCount.value++
    console.log(value)
  }
</script>
```

**Comment by elkofy** [comment]
I tried to fix this bug but I think there is a major problem for resolving this issue.
To put it simply the value inside of the input has to be empty when the component is out of focus. So the search value gets reset when out of focus and since the search value is bind to the v-model you can't really change it without reworking how the component works.
This is just what I figured out when working on it.
I had the intention to prevent the event to be emitted when the value change but i can't get it to work.

**Comment by augustoeliezer** [comment]
Two or more searches fixed in `Vuetify@3.6.4`

### Last doubt
On choose one item in list it shoulded trigger the search event?

I think it should be when typing
